addons/stock/models/res_company.py

```diff
@@ -42,6 +42,10 @@ def _default_confirmation_mail_template(self):
         help="""Day of the month when the annual inventory should occur. If zero or negative, then the first day of the month will be selected instead.
         If greater than the last day of a month, then the last day of the month will be selected instead.""")
 
+    # used for sending stock text confirmation
+    stock_text_confirmation = fields.Boolean("Stock Text Confirmation", default=True)
```

Comment:
Should be False here, and set to True in sub modules. As they are auto install it should be activated by default, but with stock only it should not try to force install sms :) .